spec/features/user_edits_bathroom_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+# Acceptance Criteria
+# [x] I can only see the edit button if I own the bathroom
+# [x] There is an edit button on the show page
+# [x] I can click the button to edit my bathroom
+# [x] I am linked to a page to edit my selected bathroom
+# [x] I am notified when I edit my bathroom
+
+feature "As an authenticated user
+I want to edit a bathroom
+so that bathroom has relevant information" do
+  scenario 'User edits a bathroom' do
+    @user = FactoryGirl.create(:user)
+    @bathroom = Bathroom.create!(
```

Comment:
If you're using FactoryGirl for user, you should also make a factory for bathroom to save yourself a lot of work!